non_ocds_collect/requirements.in

```diff
@@ -0,0 +1,7 @@
+psycopg2
+requests
+scrapy
+scrapyd
+scrapyd-client
```

Comment:
Not sure if we are deploying Scrapyd, or just manually running scrapy?

Reply:
You are right, we don't need Scrapyd